documentation/shared/attributes.adoc

```diff
@@ -109,10 +102,8 @@
 :OperatorLifecycleManager: link:https://github.com/operator-framework/operator-lifecycle-manager[Operator Lifecycle Manager^]
 :OLMOperatorDocs: link:https://olm.operatorframework.io/docs/[Operator Lifecycle Manager documentation^]
 :OpenTracingHome: link:https://opentracing.io/[OpenTracing^]
-:OpenTelemetryHome: link:https://opentelemetry.io/[OpenTelemetry^]
 :JaegerHome: link:https://www.jaegertracing.io/[Jaeger^]
 :JaegerArch: link:https://www.jaegertracing.io/docs/1.18/architecture/[Jaeger architecure^]
-:JaegerArch: link:https://www.jaegertracing.io/docs/1.18/architecture/[Jaeger architecure^]
```

Comment:
1.29?

Reply:
I guess if we want to update the links we should maybe keep it for separate PR. I did not focused on them in any way.

Reply:
_latest_ reverts to the latest version number. we can use that. There are a few jaeger links that can be replaced with this attribute. I'll make that change and replace with _latest_